widgets/events/src/EventsTable.tsx

```diff
@@ -131,37 +136,49 @@ export default class EventsTable extends React.Component {
                         label="Blueprint"
                         name="blueprint_id"
                         show={
-                            _.size(data.blueprintId) !== 1 &&
-                            _.size(data.deploymentId) !== 1 &&
-                            _.size(data.nodeInstanceId) !== 1 &&
-                            _.size(data.executionId) !== 1 &&
+                            !this.sOneElementLike(data.blueprintId) &&
```

Comment:
Looks like a typo to me :smile: 

```suggestion
                            !this.isOneElementLike(data.blueprintId) &&
```

This applies to all calls for `isOneElementLike` :slightly_smiling_face: